tests/integration/jupyter_tests.rs

```diff
@@ -517,7 +519,7 @@ async fn jupyter_kernel_info() -> Result<()> {
         "mimetype": "text/x.typescript",
         "file_extension": ".ts",
         "pygments_lexer": "typescript",
-        "nb_converter": "script"
+        "nbconvert_exporter": "script"
```

Comment:
Is this something we should integration test?

Reply:
This now gets handled by using a proper struct that serde handles.

Reply:
I agree this probably shouldn't be an integration test. This isn't testing much of anything anymore.